src/bootstrap/install.rs

```diff
@@ -66,13 +66,15 @@ fn install_sh(
     build.info(&format!("Install {} stage{} ({:?})", package, stage, host));
 
     let prefix_default = PathBuf::from("/usr/local");
-    let sysconfdir_default = PathBuf::from("/etc");
+    let sysconfdir_default = PathBuf::from("etc");
```

Comment:
Er, no, the previous default was correct I think? In that normally you want to have the sysconfdir be `/etc` and the rest of the dirs be relative to `/usr/local`. The confusing part of this is that `Path::new("/usr/local").join("/etc")` evaluates to `/etc` which is somewhat unexpected, though correct.

Reply:
Oh, wow - that is very surprising, but of course, you are right. https://play.rust-lang.org/?gist=9fca8ef12a8ac532ceaa12383983c62f&version=stable

Fixed again!